go/libkb/active_device.go

```diff
@@ -91,6 +91,12 @@ func (a *ActiveDevice) internalUpdateUIDDeviceID(acct *Account, uid keybase1.UID
 		return errors.New("ActiveDevice.set funcs must be called from inside a LoginState account request")
 	}
 	if a.uid.IsNil() && a.deviceID.IsNil() {
+		/*
```

Comment:
No, but that's what I had in there to try to find a test that could reproduce it.